diff -urN glade3-3.2.0/data/glade-3.desktop.in.in glade3-3.2.0.new/data/glade-3.desktop.in.in
--- glade3-3.2.0/data/glade-3.desktop.in.in	2007-03-11 18:13:11.000000000 +0100
+++ glade3-3.2.0.new/data/glade-3.desktop.in.in	2007-03-13 19:50:51.000000000 +0100
@@ -8,7 +8,7 @@
 StartupNotify=true
 Type=Application
 Icon=glade-3
-Categories=GTK;Application;Development;GUIDesigner;
+Categories=GNOME;GTK;Development;GUIDesigner;
 MimeType=application/x-glade;
 X-GNOME-DocPath=glade/glade.xml
 X-GNOME-Bugzilla-Bugzilla=GNOME
