diff -urN glade3-3.1.4/glade-3.desktop.in glade3-3.1.4.new/glade-3.desktop.in
--- glade3-3.1.4/glade-3.desktop.in	2006-12-07 18:44:09.000000000 +0100
+++ glade3-3.1.4.new/glade-3.desktop.in	2007-02-01 12:50:51.971414723 +0100
@@ -8,7 +8,7 @@
 StartupNotify=true
 Type=Application
 Icon=glade-3
-Categories=GTK;Application;Development;
+Categories=GTK;GNOME;Development;GUIDesigner;
 MimeType=application/x-glade;
 X-GNOME-DocPath=glade/glade.xml
 X-GNOME-Bugzilla-Bugzilla=GNOME
