--- glade3-3.6.7/configure.ac.wiget	2010-12-31 15:33:50.326562908 +0100
+++ glade3-3.6.7/configure.ac	2010-12-31 15:51:21.633744170 +0100
@@ -249,6 +249,9 @@ AC_SUBST([AM_CFLAGS])
 AC_SUBST([AM_CXXFLAGS])
 AC_SUBST([AM_LDFLAGS])
 
+LIB="$(basename $libdir)"
+AC_DEFINE_UNQUOTED([LIB],["$LIB"],[libdir basename])
+
 AC_CONFIG_FILES([
 Makefile
 data/gladeui-1.0.pc
--- glade3-3.6.7/gladeui/glade-utils.c.wiget	2010-12-31 15:30:35.934282540 +0100
+++ glade3-3.6.7/gladeui/glade-utils.c	2010-12-31 15:31:41.439362633 +0100
@@ -1473,7 +1473,7 @@ GModule *
 glade_util_load_library (const gchar *library_name)
 {
 	GModule      *module = NULL;
-	const gchar  *default_paths[] = { glade_app_get_modules_dir (), "/lib", "/usr/lib", "/usr/local/lib", NULL };
+	const gchar  *default_paths[] = { glade_app_get_modules_dir (), "/" LIB, "/usr/" LIB, "/usr/local/" LIB, NULL };
 	const gchar  *search_path;
 	gchar       **split;
 	gint          i;
