--- glade3-3.8.1/configure.ac.wiget	2011-10-11 15:21:34.643197406 +0200
+++ glade3-3.8.1/configure.ac	2011-10-11 15:28:48.139676382 +0200
@@ -254,6 +254,9 @@ AC_SUBST([AM_CFLAGS])
 AC_SUBST([AM_CXXFLAGS])
 AC_SUBST([AM_LDFLAGS])
 
+LIB="$(basename $libdir)"
+AC_DEFINE_UNQUOTED([LIB],["$LIB"],[libdir basename])
+
 AC_CONFIG_FILES([
 Makefile
 data/gladeui-1.0.pc
--- glade3-3.8.1/gladeui/glade-utils.c.wiget	2011-10-11 15:20:40.998190129 +0200
+++ glade3-3.8.1/gladeui/glade-utils.c	2011-10-11 15:21:08.850662304 +0200
@@ -1519,9 +1519,9 @@ glade_util_load_library (const gchar *li
 {
 	gchar        *default_paths[] = { (gchar *)glade_app_get_modules_dir (), 
 					  NULL, /* <-- dynamically allocated */ 
-					  "/lib", 
-					  "/usr/lib", 
-					  "/usr/local/lib", 
+					  "/" LIB, 
+					  "/usr/" LIB, 
+					  "/usr/local/" LIB, 
 					  NULL };
 
 	GModule      *module = NULL;
